File: platform/sdk/nodejs/examples/graph/graph-020_proj_attr_in.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projAttrIn("node-imp-1", { score: 99, level: "gold" });
if (!result.imported || result.imported < 1) process.exit(1);
console.log("XJ-OK: graph-020_proj_attr_in");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-020_proj_attr_in.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projAttrIn("node-imp-1", {"score": 99, "level": "gold"})
if not result["imported"] or result["imported"] < 1:
    sys.exit(1)
print("XJ-OK: graph-020_proj_attr_in")
sys.exit(0)
